packages/legend-art/src/components/panel/PanelExplorer.tsx

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright (c) 2020-present, Goldman Sachs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { clsx } from 'clsx';
+
+export const PanelExplorer: React.FC<{
```

Comment:
This is really `PanelExplorerItem`, I'm not really digging `explorer`, but I can't come up a better term for now, so we can settle with it.